src/reducers/publish.js

```diff
@@ -138,6 +138,7 @@ const generation: Reducer<number> = (state = 0, action) => {
     case 'SANITIZED_PROFILE_PUBLISHED':
     case 'UPLOAD_ABORTED':
     case 'UPLOAD_FAILED':
+    case 'HIDE_STALE_PROFILE':
```

Comment:
I believe there's a potential race where a user could abort and start again while `hideStaleProfile` happens (this lasts 300ms). Obviously this is very unlikely (yes, canceling is possible but I doubt a user will be able to cancel and start a new upload in 300ms), but I hit this while testing and this made intermittent tests.

Reply:
Makes sense, async is hard.